src/hooks/index.js

```diff
@@ -489,147 +578,183 @@ exports.mapPayload = (context) => {
 		get() {
 			if (!isProductionMode) {
 				logger.warning(
-					'reading params.payload is DEPRECATED, please use params.authentication.payload instead!'
-					+ ` path: ${context.path} method: ${context.method}`,
+					'reading params.payload is DEPRECATED, please use params.authentication.payload instead!' +
+						` path: ${context.path} method: ${context.method}`,
 				);
 			}
 			return (context.params.authentication || {}).payload;
 		},
 		set(v) {
 			if (!isProductionMode) {
 				logger.warning(
-					'writing params.payload is DEPRECATED, please use params.authentication.payload instead!'
-					+ `path: ${context.path} method: ${context.method}`,
+					'writing params.payload is DEPRECATED, please use params.authentication.payload instead!' +
+						`path: ${context.path} method: ${context.method}`,
 				);
 			}
-			if (!context.params.authentication) context.params.authentication = {};
+			if (!context.params.authentication)
+				context.params.authentication = {};
```

Comment:
Codacy found an issue: [Expected no linebreak before this statement.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5994445)